### DISCUSSION TITLE
Swaylock

### DISCUSSION THREAD

**Discussion by jpx40** [prasanthrangan/hyprdots#303] [Q&A]
Hey, how is Swaylock configured, or how can I disable it temporarily? I have the issue that my Controller isn't recognized as input during gaming and the screens locks itself after half an hour.

**Comment by prasanthrangan** [comment]
Hi, the `swayidle` timeout is configured in `~/.config/hypr/userprefs.conf`

you can use `idle_inhibitor` in waybar to disable it - 
![image](https://github.com/prasanthrangan/hyprdots/assets/106020512/ae34de7d-7f48-4c8c-85bf-80137511cf2e)

**Reply by jpx40** [reply]
Thank you

**Comment by kRHYME7** [comment]
You can try  [this ](https://github.com/ErikReider/SwayAudioIdleInhibit) also
It works on background and with the waybar module too.

**Reply by jpx40** [reply]
Thank you i will try  to add it

**Reply by jpx40** [reply]
@prasanthrangan can we add this to the repo?

**Reply by jpx40** [reply]
Do have to run this instead of swayidle or should i run it parallel

**Reply by prasanthrangan** [reply]
Ya sure, I'll add this

**Reply by kRHYME7** [reply]
We should run this parallel with swayidle

``` 
exec-once = sway-audio-idle-inhibit 
```

**Comment by jpx40** [comment]
> You can try [this ](https://github.com/ErikReider/SwayAudioIdleInhibit) also It works on background and with the waybar module too.
